src/features/reports/readme.md

```diff
@@ -0,0 +1,86 @@
+## Reports
+
+This feature is responsible for presenting a list of Kontur reports and browsing them.
+Each report has a link, a description, date of latest update and report's content represented as a table. Originally those reports are stored as `.csv` tables
+
+### How to use
+
+There are 2 separate root components for that feature.
+`<ReportsList />` shows the list of reports, their brief description and general description of reports section. It also allows to navigate to the report by clicking on its card.
+`<ReportInfo />` provides whole info about the given report, taking it's id from the link.
+Considering that it's logical to run those parts as different views(on the different routes).
+Here's expample of adding general `<ReportsList />`
+
+```ts
+import { ReportsList } from '~features/reports/components/ReportsList/ReportsList';
+import { goTo } from './yourRouterFunction';
+
+function GeneralReportsView({ goToReport }: { goToReport: (id: string) => void }) {
+  return (
+    <div>
+      <ReportsList goToReport={goToReport} />
+    </div>
+  );
+}
+
+const appRoutesConfig = [
+  {
+    title: 'All reports list',
+    view: <GeneralReportsView goToReport={(id) => goTo(`reportsPath/${id}`)} />,
+  },
+];
+```
+
+And here's how we add a single report's `<ReportInfo />`
+
+```ts
+import { ReportInfo } from '~features/reports/components/ReportInfo/ReportInfo';
+
+function ReportFullInfoPage() {
+  return <ReportInfo />;
+}
+
+// so to the routes config we would add following
+appRoutesConfig.push(
+  {
+    title: 'Single report info',
+    view: <ReportFullInfoPage } />,
+    parentRoute: 'reportsPath',
+    paramsIndicator: ':reportId'
+    // the latter is important because ReportInfo get's id th
+  }
+)
+```
+
+You have to tell your router that your searching params is "reportId" because the component gets the id the following way:
+
+```ts
+import { useParams } from 'react-router';
+
+function ReportInfo() {
+  const { reportId } = useParams();
+  // ...
+}
+```
+
+### How it works
+
+#### Data model
+
+There are 2 dynamic configs we receive from the back-end.
+
+1. Let's call it `"Reports List"`. It's an array of available reports containing all the info about each one of them. On of it's properties is the link to `.csv` file. All the keys stored in `./atoms/reportsAtom.ts` `Report` type.
```

Comment:
`"Reports List"` - array of available reports containing all info about each of them including link to `.csv` file with report data.